scripts/nesi_scripts/adv_im/clean_adv_im_tmps.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Is this script nesi specific? If not there should be a bash_scripts folder to put the agnostic non-python scripts in